templates/monitoring/jobs/openshift_monitoring_federation.yaml

```diff
@@ -9,7 +9,8 @@
   metrics_path: /federate
   params:
     match[]:
-    - '{{ "{" }} endpoint="https-metrics" {{ "}" }}'
+    - 'kubelet_volume_stats_used_bytes{endpoint="https-metrics",namespace=~"openshift-.*"}'
```

Comment:
@R-Lawton the namespace regex is too restrictive for on OSD4. It's fine for OSD3 where RHMI namespaces are prefixed with `openshift-`. However on OSD4 they are prefixed with `redhat-rhmi-`.
Based on that, I'd suggest this regex be changed to `redhat-rhmi-`.

@philbrookes Does that seem correct to you for any volumes we'd be interested in on OSD4?

Reply:
Cool ill change that and see does it change the graph

Reply:
yes the prefix in OSD is ```redhat-rhmi-*```